lib/zewo.rb

```diff
@@ -104,20 +94,16 @@ def build_dependencies
         dependency_repos = File.read(dir('Package.swift')).scan(/(?<=Zewo\/|SwiftX\/|VeniceX\/|paulofaria\/)(.*?)(?=\.git)/).map(&:first)
 
         # here we should do something like dependency_repos -= ['CMySQL', 'COpenSSL', 'CLibpq']
-
+        dependency_repos = File.read(dir('Package.swift')).scan(%r{/(?<=Zewo\/)(.*?)(?=\.git)/}).map(&:first)
```

Comment:
I think this line shouldn't be here. The comment above can be deleted as well. It was solved somewhere else.